permits/templates/permits/archived_permit_request_toolbox_header.html

```diff
@@ -0,0 +1,15 @@
+{% load static %}
+{% load bootstrap4 %}
+{% load i18n %}
+
+<div class="d-flex flex-row center-table">
+  <div class="p-2 permits-action-btns">
+      <form id="download-form" method="get" action="{% url "permits:archived_permit_request_bulk_download" %}">
+        {% csrf_token %}
+        <button class="btn btn-outline-primary" type="submit">
+          <i class="fa fa-download"></i>
+          {% trans "Télécharger" %}
```

Comment:
Don't display the button if no permissions  ?

Reply:
Not possible, since all the users that can view this page can download archives